### ISSUE TITLE
Render differences between Ubuntu and Windows

### ISSUE THREAD

**Issue by reinaldoborin** [wkhtmltopdf/wkhtmltopdf#5024]
Hi, I am having a problem at Windows.

My .pdf style works fine at Ubuntu.

But on Windows, the style of pdf isnt equal.

For example, my grid works well, table too, but the H2 doenst keep the color or size.

Already tried: css inline, css link, css cdn.

**Comment by PhilterPaper** [comment]
Are you saying that, for the _same_ HTML input, wkHTMLtoPDF running on Windows produces different PDFs than running on Ubuntu? If so, what is different between the two -- wkHTMLtoPDF version? You built one and used a prebuilt binary for the other? Or are you saying that you produce one PDF file (on either Windows or Ubuntu), and it has problems displaying correctly on Windows? If so, and the PDF was produced on Ubuntu, could it have been corrupted while transferring it to Windows?

**Comment by AimanMRT** [comment]
Yeah..already set path as windows..got wkhtmltopdf ddnt return any data

**Comment by daniberv** [comment]
I can confirm the issue but for Debian and Windows. We're using 0.12.5 version (because for some reason there is no opportunity to use 0.12.6 on Debian). Generated PDFs are different slightly. The main difference is zoom. PDF generated on Windows looks slightly increased than PDF generated on Debian.
DPI changing has no effect.
Viewport settings have no effect.
The same HTML was used.

**Comment by reinaldoborin** [comment]
I discovered the problem.

Was css variables: "var --(color-primary)",  maybe the component dont render on windows.
